core/network.py

```diff
@@ -173,8 +173,8 @@ def new_network_events(configuration):
 
     # Display filter to be applied to the Live Captured network traffic
     display_filter = ' and '.join(['ip.src!={0} and ip.dst!={0}'.format(_) for _ in ignore_ip_addresses])
-    display_filter += ' and ' if ignore_ip_addresses else ""
-    display_filter += ' and '.join(['ip.src!={0} and ip.dst!={0}'.format(_) for _ in ignore_ports])
+    display_filter += ' and ' if ignore_ports else ""
```

Comment:
this should be `ignore_ip_addresses` because it's a seperator if ignore_ip_addresses exist.

Reply:
but this is a separator between `ignore_ip_addresses` and `ignore_ports`. If there are no ports to be ignored `and` should not be there at the end.

Reply:
good point, maybe use `display_filter += ' and ' if ignore_ip_addresses and ignore_ports  else ""`